test/unit/utils/token-verifier.spec.ts

```diff
@@ -30,10 +30,12 @@ import LegacyFirebaseTokenGenerator = require('firebase-token-generator');
 import * as mocks from '../../resources/mocks';
 import { FirebaseTokenGenerator, ServiceAccountSigner } from '../../../src/auth/token-generator';
 import * as verifier from '../../../src/utils/token-verifier';
-import * as verifierUtil from '../../../src/auth/token-verifier-util';
+import {
```

Comment:
Sorry if I wasn't clear. But I didn't mean moving this entire file into utils. Some tests in this file are still very auth-specific, and therefore should be in auth. But the test cases that specifically test for the `utils.token-verifier`, should be moved into a new test module (like the `invalidErrorTypes` test cases).

I think a good rule of thumb is if a test requires an import from `auth`, then it probably belongs in `auth`.